HTTP_Engine/Compute/GetRequest.cs

```diff
@@ -31,8 +31,8 @@ public static partial class Compute
         /***************************************************/
         /**** Private  Methods                          ****/
         /***************************************************/
-
-        private static string GetRequest(string uri)
+        [System.Obsolete("GetRequest is deprecated, pit does not comply with BHoM Adaptor Push and Pull protocols")]
```

Comment:
Can you use `BH.oM.Reflection.Attributes.Deprecated` instead of `System.Obsolete?

Reply:
Yep no problem
``[Deprecated("3.1", "Does not comply with BHoM Adaptor Push and Pull protocols")]``
fromVerision = 3.1?

Reply:
Yep! Correct!